src/renderer/components/AssetDistribution/Row.js

```diff
@@ -92,8 +93,13 @@ const Row = ({ item: { currency, amount, distribution }, isVisible }: Props) =>
   const color = getCurrencyColor(currency, theme.colors.palette.background.paper);
   const percentage = (Math.floor(distribution * 10000) / 100).toFixed(2);
   const icon = <CryptoCurrencyIcon currency={currency} size={16} />;
+  const onClick = useCallback(() => {
+    track("Page Potfolio Asset allocation", { currencyName: currency.name });
```

Comment:
we shouldn't do this on click. it should be a `<TrackPage` (or Screen, whatever naming^^) on the /asset/* screen itself.